src/main/java/org/eclipse/yasson/internal/AnnotationIntrospector.java

```diff
@@ -82,13 +82,13 @@
 public class AnnotationIntrospector {
 
     private final JsonbContext jsonbContext;
+    private final ConstructorPropertiesAnnotationIntrospector constructorPropertiesIntrospector;
 
     /**
      * Annotations to report exception when used in combination with {@link JsonbTransient}
      */
-    public static final List<Class<? extends Annotation>> TRANSIENT_INCOMPATIBLE =
```

Comment:
Lots of formatting changes in this class. It would be best to avoid it since it makes PR and further GIT changes reading unnecessary challenging.

Reply:
sorry, formatting is reset